packages/core/src/App/Containers/Layout/header/header.jsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
-import { PlatformContext } from '@deriv/shared';
+import { withRouter } from 'react-router-dom';
+import { PlatformContext, getDashboardPlatformHeader } from '@deriv/shared';
+import { connect } from 'Stores/connect';
 import DefaultHeader from './default-header.jsx';
+import DashboardPlatformHeader from './dashboard-platform-header.jsx';
 import DashboardHeader from './dashboard-header.jsx';
 
-const Header = () => {
+const Header = ({ app_routing_history }) => {
     const { is_dashboard } = React.useContext(PlatformContext);
-    return is_dashboard ? <DashboardHeader /> : <DefaultHeader />;
+
+    if (is_dashboard) {
+        if (getDashboardPlatformHeader(app_routing_history)) {
```

Comment:
I think we can just check for `!isDashboard()` here. In this way, we don't have to connect this component with the store and router

Reply:
@balakrishna-binary the problem is, we have 2 headers in case of `is_dashboard===true` (one for platforms and one for other sections), and for that part I used the store and router.

Reply:
So, we show different headers based on the current path if `is_dashboard` is `true`. Why do we need `app_routing_history` for this? 
Is it possible to show different headers based on the current path (`window.location.pathname`)?

Reply:
@balakrishna-binary Base on my previous implementation I was using `getDashboardPlatformHeader` which needed `app_routing_history` to check if the routing is from the platform or not, but base on your bellow comment about omitting `/dashboard` I change the implementation and there is no need to this.